metadata.json

```diff
@@ -1,14 +1,23 @@
 {
-	"name": "cristifalcas-journald",
-	"author": "Cristian Falcas",
-	"version": "0.6.1",
-	"source": "git@github.com:cristifalcas/puppet-journald.git",
-	"project_page": "https://github.com/cristifalcas/puppet-journald",
-	"issues_url": "https://github.com/cristifalcas/puppet-journald/issues",
+	"name": "simp-journald",
+	"author": "SIMP Team",
+	"version": "1.0.0",
```

Comment:
Do we want this to be 1.0.0 or 0.7.0?

Reply:
I made a breaking change to the API, if we deprecate the other module in favor of this one, it should be a major bump